### DISCUSSION TITLE
Monaco Treemending script

### DISCUSSION THREAD

**Discussion by samirzubi-db** [CodinGame/monaco-vscode-api#125] [Q&A] [answered]
Greetings,

I wanted to know more about future plans for the monaco-treemending script. Due to the script changing the files of another package it causes issues down the line that need to be adhered to (I saw a couple of issues pop up in the last weeks). In my example it causes issues when the vscode files are readonly due to the project setup. I wanted to know if there is there a plan to move away from this approach? 

Thanks!

**Comment by CGNonofr** [comment]
There is currently no plan to move away from it as we're not aware of any better option

**Reply by CGNonofr** [reply, marked as answer]
Another approach was introduced, there is now a @codingame/monaco-editor-treemended package that can be used as a replacement of `monaco-editor` and which is already treemended
